react/Field/styles.styl

```diff
@@ -12,3 +12,8 @@
     top -3rem
     right 0
     cursor pointer
+
+.o-side
```

Comment:
Last feedback: Maybe semantically it could be better to have only an `o-side` class. What do you think ?  @ptbrowne ? @GoOz ?

Reply:
ok to switch and place o-side on the main class and o-field-input-action extends this